frontend/subrequests/describe.go

```diff
@@ -0,0 +1,15 @@
+package subrequests
+
+const RequestSubrequestsDescribe = "frontend.subrequests.describe"
+
+var SubrequestsDescribeDefinition = Request{
+	Name:        RequestSubrequestsDescribe,
+	Version:     "1.0.0",
+	Type:        TypeRPC,
```

Comment:
What other types are you envisioning?

Reply:
Not sure. Maybe something that works in combination with exec support.